### PR TITLE
Implement composite mode in GrayToColor

### PR DESCRIPTION
**Pull request by LeeKamentsky**
This lets the user specify an arbitrary number of input images and colors for each. Each grayscale image is converted to its chosen color and then the set of color images are composited together by a simple addition.

### CODE REVIEW

**Comment by braymp** [comment]
Here is an image from CPA which does a good job of color composition IMHO:
image

With the GrayToColor mods:
image

Pretty good. Might be good to add settings for relative weights as well, as the RGB option does.

**Comment by LeeKamentsky** [comment]
If you are clever, you can just pick darker colors to adjust the relative weights (but the scale is limited because each color only has a 0-255 range). But I agree that having the relative weights is a more obvious solution and we are likely to have users who didn't figure out the work-around asking for the feature. I will mark the pull request "WIP" and add the weights unless someone objects.

**Comment by LeeKamentsky** [comment]
@braymp added weights.

**Comment by braymp** [comment]
:+1:

File: cellprofiler/modules/graytocolor.py [hunk 1]
```diff
@@ -35,19 +35,30 @@
...
+OFF_STACK_CHANNEL_COUNT_V3 = 16
+OFF_STACK_CHANNEL_COUNT = 16
+
 SCHEME_RGB = "RGB"
 SCHEME_CMYK = "CMYK"
 SCHEME_STACK = "Stack"
+SCHEME_COMPOSITE = "Composite"
 LEAVE_THIS_BLACK = "Leave this black"
 
+DEFAULT_COLORS = [
```
Comment (0x00B1) [inline cellprofiler/modules/graytocolor.py]: Cute! :dancer:

**Pull request merged**